src/website/app/pages/Typography/index.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import colorable from 'colorable';
+import { createStyledComponent } from '../../../../utils';
+import Markdown from '../../Markdown';
+import {
+  Table as _Table,
+  TableCell,
+  TableHeaderCell,
+  TableRow
+} from '../../Table';
+import FontDemo from './FontDemo';
+import content from './typography.md';
+import examples from './examples';
+
+const a11yColor = color => {
+  const a11y = colorable({
+    main: color,
+    white: '#fff',
+    black: '#000',
+    compact: true
+  });
+
+  const whiteContrast = a11y[0].combinations[0].contrast;
+  const blackContrast = a11y[0].combinations[1].contrast;
+
+  return whiteContrast > blackContrast ? 'white' : 'black';
+};
+
+const styles = {
+  colorValue: ({ theme, color }) => {
+    return {
+      backgroundColor: theme[color],
+      borderRadius: theme.borderRadius_1,
+      color: a11yColor(theme[color]),
+      fontFamily: theme.fontFamily_monospace,
+      padding: `${theme.spacing_half} ${theme.spacing_single}`
+    };
+  },
+  exampleCell: {
+    // the focused link examples are cut off when the container is overflow: auto
+    paddingLeft: 3,
+    '& > a': {
+      // not sure why this is needed. cursor: 'auto' is rendering the i-beam cursor
+      cursor: 'pointer'
+    }
+  },
+  overflowContainer: ({ theme }) => ({
+    overflowX: 'auto',
+    // the outline-offset on the focused link example does not count towards the box model
+    // and gets cut off at the bottom when the text wraps
+    paddingBottom: theme.spacing_single
```

Comment:
This isn't necessary; you aren't rendering Link.

Reply:
addressed